roles/openshift_imagestreamtags/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Ensure openshift project exists
+  oc_project:
+    name: openshift
+    state: present
+    node_selector:
+    - ""
+
+- name: Tag image into an image stream in the openshift namespace
+  command: >
+    {{ openshift_client_binary }} tag --source=docker {{oreg_url}} --config={{ openshift.common.config_base }}/master/admin.kubeconfig openshift/{{ openshift_ist }}
```

Comment:
oreg_url normally takes the form:
`registry.hostname.com/openshift3/ose-${component}:${version}`

(ie it's normally a template, not a specific image spec)

I don't see how you can use it in this way here?

In general if we're going to do this at all, i'd expect this to take two inputs:

1) an array of image sources
2) an array of ISTs to tag those sources as

Reply:
We can certainly introduce a new environment variable if straying from the typical format of `oreg_url` is less desirable than introducing another variable.

@sdodson - any opinions there?

Reply:
the bad part about reusing oreg_url but requiring a different format/value is it means someone has to edit their inventory file in between running playbooks.  That seems error prone.

Reply:
ok i'm convinced ... will introduce new var